ToDoApp/ToDoApp/ToDoAppApp.swift

```diff
@@ -11,7 +11,8 @@ import SwiftUI
 struct ToDoAppApp: App {
     var body: some Scene {
         WindowGroup {
-            ContentView()
+//            ContentView()
```

Comment:
let's don't leave useless comments in the code

Reply:
This has been fixed.